dips/dip-10.md

```diff
@@ -13,47 +14,37 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 This DIP describes Diem ID - the human-readable identifier for user accounts.   
 
-In addition to describing the format of the Diem ID, this DIP also describes the off-chain protocol for the peer-2-peer payment with Diem ID.
-
 # Motivation
-Diem ID provides a convenient way to identify a user's account within a VASP. There are two main use cases for Diem ID:
-* For peer-2-peer payments, Diem ID allows a user to provide a human-readable ‘identifier’ to the sender. Basically, instead of sending a [dip-5 address](https://github.com/diem/dip/blob/master/dips/dip-5.md) (that needs to be refreshed for each payment, ideally), the receiver can just share their Diem ID once. In this case Diem ID plays the role of an email address for payments. Benefits for this use case:
-  * The Diem ID resolution process can facilitate the exchange of rich information(e.g. nicknames, profile picture, etc.) about both the sender and receiver of the payment
-  * Diem ID provides better privacy compared to using sub-addresses, because Diem ID is not shared on the chain and will always use a single-use identifier for the on-chain ID
+Diem ID provides a convenient way to identify a user's account within a VASP.
+* For peer-2-peer payments, Diem ID allows a user to provide a human-readable ‘identifier’ to the sender. Instead of sending a [dip-5 address](https://github.com/diem/dip/blob/master/dips/dip-5.md) (that ideally needs to be refreshed for each payment), the receiver can just share their Diem ID once. In this case Diem ID plays the role of an email address for payments. Benefits for this use case:
```

Comment:
thinking something like: allows users to exchange human-readable identifiers as both the sender and recipient of payments peer-to-peer payments